assets/README.md

```diff
@@ -0,0 +1,2 @@
+# Flare Files
+Files by [@mathis-m](https://github.com/mathis-m/) can be forked [here](https://rive.app/a/mathis-m/files/recent/all).
```

Comment:
Link the individual files and set the license to [Unlicense](https://unlicense.org/)